### PR TITLE
[prometheusexporter] Make metrics path configurable

### LINKED ISSUE #8810: make "/metrics" for  `prometheusexporter`  configurable
**Is your feature request related to a problem? Please describe.**
make "/metrics" for  `prometheusexporter`  configurable in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusexporter/prometheus.go#L75

### PR DESCRIPTION
**Pull request by gouthamve**
Fixes https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/8810

### CODE REVIEW

File: exporter/prometheusexporter/prometheus_test.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func TestPrometheusExporter(t *testing.T) {
 					"code0": "one0",
 				},
 				Endpoint:         ":8999",
+				MetricsPath:      "/metrics",
```
Comment (anuraaga) [inline exporter/prometheusexporter/prometheus_test.go:51]: Can we add a test using a different path to ensure the config is being read?

Comment (gouthamve) [reply exporter/prometheusexporter/prometheus_test.go:51]: Hi, done!

**Comment by gouthamve** [comment]
@dashpole Could you review this and remove the `stale` label?

**Review by dashpole — APPROVED** [review]
The change LGTM.  Please add the rationale to the change to the description, since the issues doesn't really cover it.  I assume they want it to be able to run multiple prometheus exporters on the same port...

**Comment by gouthamve** [comment]
> I assume they want it to be able to run multiple prometheus exporters on the same port

I don't think this lets them run two exporters on the same port. Unless they use a reverse proxy. But if they are using a reverse proxy they don't need this config option. Let me comment on the original issue to understand the use-case better.

**Pull request closed without merging**